### PR TITLE
Small changes to washboards. Now heavier but actually slightly smaller.

### PR DESCRIPTION
**Pull request by faefux**
#### Summary
Content "Changed wooden washboards a little."

#### Purpose of change

Always wondered they were a little easy to build and carry. Always felt should be much heavier and harder to build and pack.

#### Describe the solution

Changed Dimensions and weight as per: [This one Amazon example](https://www.amazon.co.uk/Petyoung-Washboard-Laundry-Clothes-Washing/dp/B08VWCRZT4).  Will pave the way for a hopefully lighter and harder to craft washboard (aluminium is my thoughts on this, personally)

#### Describe alternatives you've considered
I was going to up it's Fabrication level as it seems low, but with the new system in place of proficiency, etc. I am not so confident on what to change it to. Perhaps this could be decided between us.

#### Testing

Game ran just fine. make style-json ran just fine.

#### Additional context

Let us have a discussion on if fabrication level to make a wooden washboard (without the curved chisels, etc. that would make it easy), the time it takes, proficiencies potentially needed and tool use. The current version as below is to me highly problematic:
`  {
    "type": "recipe",
    "activity_level": "LIGHT_EXERCISE",
    "result": "washboard",
    "category": "CC_OTHER",
    "subcategory": "CSC_OTHER_TOOLS",
    "skill_used": "fabrication",
    "difficulty": 1,
    "time": "1 m",
    "autolearn": true,
   "qualities": [ { "id": "CUT", "level": 1 } ],
   "components": [ [ [ "2x4", 1 ] ] ]
  }
`
At the very least I would say chiseling tool or a higher level cut.  I would *possibly* change the 2x4 to a bigger board, though unsure, push time up MASSIVELY and add a proficiency for that kind of work (the name in English escapes me!).

Image for reference.
image

### CODE REVIEW

File: data/json/items/tool/toileteries.json [hunk 1]
```diff
@@ -230,9 +230,9 @@
     "type": "TOOL",
     "name": { "str": "washboard" },
     "description": "This is a wooden washboard.  You can use it to wash filthy clothing if it's supplied with cleansing agent.",
-    "weight": "786 g",
-    "volume": "1966 ml",
-    "longest_side": "40 cm",
+    "weight": "116 kg",
```
Comment (Maleclypse) [inline data/json/items/tool/toileteries.json:233]: I don't think this weight is correct.

Comment (faefux) [reply data/json/items/tool/toileteries.json:233]: > I don't think this weight is correct.
Oh yes, confusion when i converted back from G to KG i think. It is 1.16KG. Agh. thought it was ready to go, apparently not. I'll take anther look.

**Comment by faefux** [comment]
This is the kind of tool I would imagine you would fashion to make such a device, personally.  Essentially a little hand router more efficient than using most chisels.
image

**Comment by Night-Pryanik** [comment]
_Hand-crafted_ washboards doesn't need to have ideal curvy hollows. Rough edges made by knife are sufficient to wash clothes. No need to made recipe more difficult, require some proficiencies or some advanced tools.

**Comment by natsirt721** [comment]
I could stand to see the crafting time be cranked up, perhaps to 10 or 20 minutes. Cutting all those grooves is going to take more than a minute.

**Comment by Night-Pryanik** [comment]
Yeah, seems like I set one minute as a dummy value, but it wasn't set to a normal value ever since.

**Comment by faefux** [comment]
Great. I'll change the build speed in this PR. any ideas on a lighter metal one like we traditionally see in modern Americana?

**Comment by actual-nh** [comment]
> _Hand-crafted_ washboards doesn't need to have ideal curvy hollows. Rough edges made by knife are sufficient to wash clothes. No need to made recipe more difficult, require some proficiencies or some advanced tools.

Wouldn't rough edges catch on and potentially tear the clothes?

**Comment by Night-Pryanik** [comment]
> Wouldn't rough edges catch on and potentially tear the clothes?

The edges ain't THAT rough.
The survivor is careful and tries his best not to damage his clothes - both when crafting the washboard and when washing clothes.

**Pull request merged**